CODEOWNERS

```diff
@@ -299,8 +299,8 @@ geode-management/src/test/script/update-management-wiki.sh   @onichols-pivotal
 #------------------------------------------------------------
 # Documentation
 #------------------------------------------------------------
-#geode-book/**
-#geode-docs/**
+geode-book/**                                                @karensmolermiller @davebarnes97
+geode-docs/**
```

Comment:
you've uncomment geode-docs but not added any reviewers for it

Reply:
Got it. Thanks.